### DISCUSSION TITLE
Kali and Ubuntu Machines not launching

### DISCUSSION THREAD

**Discussion by RwdBass** [utmapp/UTM#4588] [UTM Help (macOS)] [answered]
**Describe the issue**
I've been using UTM for months and suddenly whenever I open any one of my VMs it just gets stuck on a black screen. Literally 2 days all was normally working.
I will attach debug logs and the black screens.
Please let me know asap what I can do to fix them

**Configuration**
* UTM Version: 3.2.4
* OS Version: MacOS Monterey version 12.4
* Apple M1 2020

**Debug log**
[KaliLinux_debug.log](https://github.com/utmapp/UTM/files/9261543/KaliLinux_debug.log)
[ubuntu_debug.log](https://github.com/utmapp/UTM/files/9261545/ubuntu_debug.log)

**Upload VM**
<img width="1440" alt="Kali Linux" src="https://user-images.githubusercontent.com/72977353/182902697-9d8743a7-ae65-43b7-b057-069001f8049f.png">
<img width="1440" alt="Ubuntu" src="https://user-images.githubusercontent.com/72977353/182902710-cf167727-5cba-4c93-9314-d2536ad6f4be.png">

**Comment by comes** [comment]
Same issue here. Upgrading to the latest Version `3.2.4 (58)` fixed that issue on my end.

cheers,

**Comment by jenselofsson** [comment]
I'm experiencing the exact same problem, it was literally from one boot to the next. I powered off the VM, and when I tried to start it again and get through GRUB I'm just met with a black screen with a blinking cursor.

This is with both a Kali 2022.2 machine that have been working for a month (since I installed it) as well as any new machine I'm trying to install. I've tried it with the 4.0 beta release as well as 3.2.4 of UTM but no difference. QEMU version is 7.0.0.

Any help is appreciated.

EDIT: If there are any logs I can supply, please let me know.

**Comment by ajay-mandal** [comment]
Same Issue but didn't get the solution

**Comment by ghost** [comment]
try this [#4090 ](https://github.com/utmapp/UTM/issues/4090#issuecomment-1213306172)

**Comment by jenselofsson** [comment]
@Treintillin That solution worked, thanks! Even for a already installed machine. I got dropped into an initramfs shell and had to run fsck on the disk, and then I was able to reboot into a graphical environment.

**Comment by ghost** [comment]
happy of reading you :)

**Comment by satyvm** [comment]
Heyy, I am not able to see the TYPE option in Display(there is no such option above Emulated Display card option), can anyone help me with this.
<img width="1012" alt="Screenshot 2022-11-01 at 7 30 18 AM" src="https://user-images.githubusercontent.com/75127014/199142603-471fa534-29ce-4774-8902-d12fa5e9aea0.png">

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/guides/kali/#black-screen-on-start

**Reply by satyvm** [reply]
Thanks, The issue is resolved now.
